java/src/main/java/io/crate/jdbc/sample/Controller.java

```diff
@@ -27,14 +27,30 @@
     private static final int CREATED = 201;
     private static final int OK = 200;
 
-    public Controller(final DataProvider model) {
+    Controller(final DataProvider model) {
 
         before(((request, response) -> {
             response.header("Access-Control-Allow-Origin", "*");
             response.header("Access-Control-Request-Method", "*");
```

Comment:
this header required? seems like it was a wrong header and the new one is superseding that